ibridges/__main__.py

```diff
@@ -41,8 +45,9 @@
 ibridges download "irods:~/test.txt"
 ibridges upload ~/test.txt "irods:/test"
 ibridges init
-ibridges sync ~/collection "irods:~/collection"
+ibridges sync ~/directory "irods:~/collection"
 ibridges list irods:~/collection
+ibridges mkdir irods://~/bli/bla/blubb
```

Comment:
Ha! Let's keep all the documentation to the old format for consistency.
```suggestion
ibridges mkdir "irods:~/some_collection/new_collection"
```